src/style.css

```diff
@@ -1 +1,74 @@
 /* Styles go here */
+
+h1 {
+  display: none;
+}
+
+body {
+  margin: 0;
+}
+
+header {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+
+}
```

Comment:
check out point 1 in the [checklist](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md)